gnomad/main.py

```diff
@@ -24,8 +24,9 @@
     init=['gs://cpg-reference/hail_dataproc/install_common.sh'], # what does this do ? plotting, analysis-runner
     cluster_name='gnomADv4',
     worker_machine_type = 'n1-standard-8',
-    num_workers = 2,
-    num_secondary_workers = 20,
+    num_workers = 20,
```

Comment:
This is *way* more expensive than preemptible workers. Please don't use non-preemptibles without a good reason.

Reply:
good point, will explore the option of limiting the number of workers to 2 but bumping the disk size